components/sup/src/event.rs

```diff
@@ -233,14 +235,17 @@ pub fn health_check(metadata: ServiceMetadata,
         let StandardStreams { stdout, stderr } =
             maybe_process_output.map(ProcessOutput::standard_streams)
                                 .unwrap_or_default();
+        let interval = health_check_interval.into();
+
         publish(HEALTHCHECK_SUBJECT,
                 HealthCheckEvent { service_metadata: Some(metadata),
                                    event_metadata: None,
                                    result: i32::from(health_check_result),
                                    execution: maybe_duration.map(Duration::into),
                                    exit_status,
                                    stdout,
-                                   stderr });
+                                   stderr,
+                                   interval: interval });
```

Comment:
Minor - I think you can just directly specify the interval as a param without the "interval:" prefix..

Reply:
Yeah, similar to `exit_status`, `stdout`, and `stderr`.

https://doc.rust-lang.org/book/ch05-01-defining-structs.html#using-the-field-init-shorthand-when-variables-and-fields-have-the-same-name

Reply:
As an FYI, running `clippy` locally (using `make lint` at the top level of the repo) will catch things like this.

Reply:
ty!